charts/latest/azuredisk-csi-driver/values.yaml

```diff
@@ -131,6 +133,7 @@ snapshot:
     name: csi-snapshot-controller
     replicas: 2
     podLabels: {}
+    annotations: {}
```

Comment:
could you use `podAnnotations` naming, that would be consistent with existing `podLabels`, thanks.

Reply:
I added `labels`, `annotations` and `podAnnotations`

Reply:
Thank you.